azerothcore/azerothcore-wotlk
# fix(DB/Creature) Correct order of Shandy Glossgleam dialogue

Makes Shandy say his for a Suitable Disguise in the correct order



## Changes Proposed:

This PR proposes changes to:

Order of dialogues was mixed up, causing issues such as the quest seeming like it had failed (ie. Sandy said the failure dialogue) despite the fact that there's no scripted failure event in the underlying cpp script, or asking for shirts instead of saying the second half of the scripted intro RP

## Issues Addressed:

- Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/744

## SOURCE:

The changes have been validated through:

https://www.youtube.com/watch?v=DHmeKq9t5Z0

## Tests Performed:

This PR has been:


## How to Test the Changes:



1. .quest add 24556 (horde) or .quest add 20438 (ally)
2. .go creature 1790
3. initiate the quest RP and see the dialogue mismatch

## Known Issues and TODO List:


- [X] The failure event is missing


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
